index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+    
+  <!-- build:css css/main.css -->
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  <!-- endbuild -->
+
+  <title>Dangelo Arrivillaga Portfolio</title>
+</head>
+
+<body>
+  <div class="contenedor">
+    <header>
+      <div class="container">
+        <div class="h-block">
+          <p class="h2-style">Portfolio</p>
+        </div>
+        <div class="h-block container fd-rr mr">
+          <div class="c-container">
+            <hr class="b-hamburger">
+            <hr class="b-hamburger">
+            <hr class="b-hamburger">
+          </div>
+        </div>
+      </div>
+    </header>
+    <div class="c-container" id="headline">
+      <h1 class="tx-center">Dangelo Arrivillag</h1>
+      <h2 class="tx-center">Full-Stack Software Developer</h2>
+      <p class="tx-center" id="p-headline">
+          I can help you build a product , feature or website Look through 
+          some of my work and experience! If you like what you see and have 
+          a project you need coded,  do not hestiate to contact me.
+      </p>
+      <div class="blur-circle"></div>
+    </div>
```

Comment:
- [ ] Great job implementing the markup, however, it is best to follow [HTML & CSS best practices,](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) one of which is to use HTML5 semantic tags (e.g., header, nav, section) over `div`s.